src/Symfony/Component/Process/Process.php

```diff
@@ -465,6 +465,11 @@ public function waitUntil(callable $callback)
                     $this->fallbackStatus['exitcode'] = (int) $data;
                 }
             }
+            if ($this->isRunning()) {
```

Comment:
what about writting in this way:
```
if ($wait && !$this->isRunning()) {
    return false;
}
```
+ make the function return a boolean (type-hint+return true at the end)

Reply:
Hmm this might change function's specification but doable

Reply:
the method is new so we can still do it. I'd makes sense to me.

Reply:
Ok updated to return a boolean.